src/include/txn_inline.h

```diff
@@ -229,6 +229,21 @@ __txn_next_op(WT_SESSION_IMPL *session, WT_TXN_OP **opp)
     return (0);
 }
 
+/*
+ * __txn_swap_snapshot --
+ *     Swap the snapshot pointers.
+ */
+static inline void
+__txn_swap_snapshot(uint64_t **snap_a, uint64_t **snap_b)
+{
+    uint64_t *temp;
+    WT_UNUSED(temp);
```

Comment:
Why WT_UNUSED is needed?

Reply:
Fixed in 1c730c4490846a7ec19c0481d4fe3199e117a403